src/ddmd/mars.d

```diff
@@ -186,67 +187,6 @@ Where:
 ", FileName.canonicalName(global.inifilename), fpic, m32mscoff, mscrtlib);
 }
 
-/// DMD-generated module `__entrypoint` where the C main resides
-extern (C++) __gshared Module entrypoint = null;
-/// Module in which the D main is
-extern (C++) __gshared Module rootHasMain = null;
```

Comment:
I had to do it in order to exclude mars.d (where main is located) from the library